components/analytics/system_analytics/system_analytics.tsx

```diff
@@ -30,10 +32,22 @@ const StatTypes = Constants.StatTypes;
 type Props = {
     isLicensed: boolean;
     stats?: Record<string, number | AnalyticsRow[]>;
+    pluginStatHandlers: GlobalState['plugins']['siteStatsHandlers'];
```

Comment:
I didn't see this pattern in any other place. I see some benefits in writing the type once and linking it from other points but I'm not 100% sure about it.

Reply:
+1 for this. We are starting to use this pattern in more places.